src/libNode/PoWProcessing.cpp

```diff
@@ -220,6 +227,28 @@ bool Node::StartPoW(const uint64_t& block_num, uint8_t ds_difficulty,
     }
   }
 
+  if (sentToDs) {
+    auto func = [this]() mutable -> void {
+      unique_lock<mutex> lk(m_mutexCVWaitDSBlock);
+      if (cv_waitDSBlock.wait_for(
+              lk,
+              chrono::seconds(NEW_NODE_SYNC_INTERVAL + POW_WINDOW_IN_SECONDS +
+                              FALLBACK_EXTRA_TIME)) == cv_status::timeout) {
```

Comment:
Do you mean the sleep should be longer than enough for triggering view change? if so should use FALLBACK_INTERVAL_WAITING instead

Reply:
The sleep must be longer than POW_WINDOW + DS block consensus

Reply:
For that purpose, should use some other constants, as this FALLBACK_EXTRA_TIME is used for different purpose